### PR TITLE
[ng] Expose tf_globals as Polymer component for interop

### PR DESCRIPTION
**Pull request by stephanwlee**
We need to be able to turn off the fake hashing in the storage module when using angular based TensorBoard.

### CODE REVIEW

File: tensorboard/components/tf_globals/globals-polymer.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```
Comment (caisq) [inline tensorboard/components/tf_globals/globals-polymer.ts]: 2020

File: tensorboard/components/tf_globals/globals-polymer.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+==============================================================================*/
+namespace tf_globals {
+  // HACK: this Polymer component allows stores to be accessible from
+  // tf-ng-tensorboard by exposing otherwise mangled smybols.
+  Polymer({
+    is: 'tf-globals',
+    _template: null, // strictTemplatePolicy requires a template (even a null one).
+    tf_globals: tf_globals,
+  });
+} // namespace tf_globals
```
Comment (caisq) [inline tensorboard/components/tf_globals/globals-polymer.ts]: style nit: put two spaces before `//`

File: tensorboard/components/tf_globals/globals-polymer.ts (hunk 1 above)
Comment (stephanwlee) [reply tensorboard/components/tf_globals/globals-polymer.ts]: but... but... commit was made in 2019! Will fix it.

File: tensorboard/components/tf_globals/globals-polymer.ts (hunk 2 above)
Comment (stephanwlee) [reply tensorboard/components/tf_globals/globals-polymer.ts]: hmm... prettier changes it back to single space. Will not fix.

**Pull request merged**